packages/test-utils/src/error.js

```diff
@@ -1,14 +1,33 @@
 import { warn } from 'shared/util'
-import { findAllInstances } from './find'
+import { findAllInstances, findAllParentInstances } from './find'
 
-function errorHandler(errorOrString, vm) {
+function errorHandler(errorOrString, vm, info) {
   const error =
     typeof errorOrString === 'object' ? errorOrString : new Error(errorOrString)
 
+  // If a user defined errorHandler was register via createLocalVue
+  // find and call the user defined errorHandler
+  const instancedErrorHandlers = findAllParentInstances(vm)
```

Comment:
maybe just move this into line `21` that already has the conditional check? A lot of the type checking is for older versions of Vue